CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [Unreleased]
```

Comment:
The changelog update is itself not necessary for this commit as the module uses the GitHub Changelog Generator - the entry in the changelog will be the PR title and placed in the appropriate section depending on the label (in this case, `backwards-incompatible`, I think, unless we're able to alias the parameter?

Reply:
Removed the changelog edit.